.github/workflows/kind_upgrade.yml

```diff
@@ -165,3 +165,34 @@ jobs:
       - name: Run test
         run: |
           ./ci/kind/test-upgrade-antrea.sh --from-version-n-minus 2 --controller-only
+
+  test-prometheus-metrics-doc:
```

Comment:
this doesn't really belong to `kind_upgrade.yml` :) But I'd be fine with `kind.yml`